Source/KSCrash/Source/KSCrash/Recording/Sentry/BSG_KSCrashSentry.h

```diff
@@ -162,7 +162,7 @@ typedef struct BSG_KSCrash_SentryContext {
 BSG_KSCrashType
 bsg_kscrashsentry_installWithContext(BSG_KSCrash_SentryContext *context,
                                      BSG_KSCrashType crashTypes,
-                                     void (*onCrash)(void));
+                                     void (*onCrash)(char, char *));
```

Comment:
The `KSCrashSentry_Tests` setup hasn't been updated to use this new signature (although it doesn't seem to have much of an effect on the overall result of the tests)